integrationtest/backend/lucene/src/test/java/org/hibernate/search/integrationtest/backend/lucene/testsupport/util/LuceneTckTestRunner.java

```diff
@@ -17,6 +17,6 @@
  * @author Gunnar Morling
  */
 @RunWith(ClasspathSuite.class)
-@ClasspathSuite.ClassnameFilters({ ".*\\.tck\\..*" })
+@ClasspathSuite.ClassnameFilters({ ".*\\.tck\\..*FieldSearchProjection.*IT" })
```

Comment:
Not sure that one was intended?

Reply:
It wasn't. I fixed it on master, thanks.